src/Facility.CodeGen.JavaScript/JavaScriptGenerator.cs

```diff
@@ -440,6 +456,14 @@ public override CodeGenOutput GenerateOutput(ServiceInfo service)
 						code.WriteLine("'TooManyRequests': 429,");
 						code.WriteLine("'InternalError': 500,");
 						code.WriteLine("'ServiceUnavailable': 503,");
+
+						httpServiceInfo.ErrorSets.ToList().ForEach(errorSetInfo =>
```

Comment:
Prefer `foreach` to `ForEach`.

Reply:
Updated with https://github.com/FacilityApi/FacilityJavaScript/pull/57/commits/4679eb1bc03c095dcd8be45c98bd3c00d95aa28b